fonduer/parser/models/phrase.py

```diff
@@ -189,12 +189,12 @@ def __repr__(self):
                 if self.col_start != self.col_end
                 else self.col_start
             )
-            return "Phrase (testDoc: '{}', Table: {}, Row: {}, Col: {}, Index: {}, Text: '{}')".format(
+            return "Phrase (Doc: '{}', Table: {}, Row: {}, Col: {}, Index: {}, Text: '{}')".format(
                 self.document.name,
                 self.table.position,
                 rows,
                 cols,
-                self.position,
+                self.phrase_num,
```

Comment:
Eliminating the redundancy of position/phrase_num.

Reply:
👍